src/pages/Category/CategoryCard.js

```diff
@@ -0,0 +1,208 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import GlobalStyles from '../../styles/GlobalStyles';
+
+function CategoryCard(props) {
+  return (
+    <>
+      <GlobalStyles />
+      <Contents>
+        {props.cardData.map(el => {
+          return (
+            <WrapCardItem key={el.id}>
+              <CardItem>
+                <CardHeader>
+                  <ProfileImg alt="profileImg" src="/images/user.png" />
+                  <ProfileName>yurim4545</ProfileName>•<Follow>팔로우</Follow>
+                </CardHeader>
+                <CardContent>
+                  <CardImg alt={el.imgAlt} src={el.imgUrl} />
+                  <Views>
+                    조회수 <span>1,225</span>
+                  </Views>
+                </CardContent>
+                <CardItemActionList>
+                  <CardAction>
+                    <CardItemAction>
+                      <ActionImg alt="좋아요" src="/images/heart.png" />
+                    </CardItemAction>
+                    <Count>8</Count>
+                  </CardAction>
+                  <CardAction>
+                    <CardItemActionAnchor>
+                      <ActionImg alt="댓글" src="/images/bubble.png" />
+                    </CardItemActionAnchor>
+                    <Count>1</Count>
+                  </CardAction>
+                </CardItemActionList>
+                <Review>{el.desc}</Review>
+                <Comment>
+                  <UserImg alt="userImg" src="/images/user.png" />
+                  <div>
+                    <UserName>yurim4545</UserName>
+                    <CommentText>
+                      푸릇한 식물과 연두색💚 침대 커버 조화가 너무 예뻐요!
+                    </CommentText>
+                  </div>
+                </Comment>
+              </CardItem>
+            </WrapCardItem>
+          );
+        })}
+      </Contents>
+    </>
+  );
+}
+
+const Contents = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  margin: auto;
+  width: 1440px;
+`;
+
+const WrapCardItem = styled.article`
+  margin-top: 30px;
+  padding: 15px;
+  -webkit-font-smoothing: antialiased;
+  letter-spacing: -0.4px;
+`;
+
+const CardItem = styled.div`
+  width: 300px;
+  padding-bottom: 40px;
+`;
+
+const CardHeader = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-bottom: 16px;
+`;
+
+const ProfileImg = styled.img`
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+`;
+
+const ProfileName = styled.span`
+  padding-left: 10px;
+  color: ${props => props.theme.fontMainBlack};
+  font-size: 18px;
+  font-weight: 500;
+`;
+
+const Follow = styled.div`
+  color: ${props => props.theme.mainBlue};
+  font-size: 18px;
+  font-weight: 500;
+`;
+
+const CardContent = styled.div`
+  width: 300px;
+  height: 300px;
+  overflow: hidden;
+`;
+
+const CardImg = styled.img`
+  width: 100%;
+  height: 100%;
+  border-radius: 6px;
+  object-fit: cover;
+  &:hover {
```

Comment:
다른 선택자들끼리 한줄 공백 넣어주세요!
```suggestion
  object-fit: cover;
  
  &:hover {
```